assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -49,7 +49,7 @@ export default function useMediaPicker({
   useEffect(() => {
     try {
       // Work around that forces default tab as upload tab.
-      wp.media.controller.Library.prototype.defaults.contentUserSetting = false;
+      global.wp.media.controller.Library.prototype.defaults.contentUserSetting = false;
```

Comment:
Can you explain why changing this to global helps here or is this a refactor?

Reply:
Just a code quality refactor to make it clear that this is a global variable. Makes testing easier too.

Note that `global` in the browser will mean `window`, so `window.wp` as expected